smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/Http/DefaultHttpClient.cs

```diff
@@ -144,20 +153,25 @@ public virtual HttpResponse Request(HttpRequest smartsheetRequest, string object
             restRequest.AddFile("file", File.ReadAllBytes(file), new FileInfo(file).Name, fileType);
             if (smartsheetRequest.Entity != null && smartsheetRequest.Entity.GetContent() != null)
             {
-                restRequest.AddParameter(objectType.ToLower(), System.Text.Encoding.Default.GetString(smartsheetRequest.Entity.Content),
-                    smartsheetRequest.Entity.ContentType, ParameterType.RequestBody);
+                //restRequest.AddParameter(objectType.ToLower(), System.Text.Encoding.Default.GetString(smartsheetRequest.Entity.Content),
+                //    smartsheetRequest.Entity.ContentType, ParameterType.RequestBody);
+                
+                BodyParameter bodyParameter = new BodyParameter(objectType.ToLower(), System.Text.Encoding.Default.GetString(smartsheetRequest.Entity.Content),
+                    smartsheetRequest.Entity.ContentType);
+                restRequest.AddParameter(bodyParameter);
             }
 
             restRequest.AlwaysMultipartFormData = true;
 
             // Set the client base Url.
-            httpClient.BaseUrl = new Uri(smartsheetRequest.Uri.GetLeftPart(UriPartial.Authority));
 
+            
+            //httpClient.Options.BaseUrl = new Uri(smartsheetRequest.Uri.GetLeftPart(UriPartial.Authority));
```

Comment:
@lifebeyondfife @stejachittineni This is why most of the tests are failing I believe. We need a way to set BaseUrl during a specific request. 

Access to BaseUrl has been removed in RestSharp 107+. Just to give context for our discussion sometime soon :)